next-tavla/app/globals.css

```diff
@@ -41,6 +41,15 @@
         --trolleybus-color: var(--colors-transport-default-bus);
         --unknown-color: var(--colors-transport-default-walk);
         --water-color: var(--colors-transport-default-ferry);
+
+        --data-visualization-azure: var(--colors-data-default-azure);
+        --data-visualization-blue: var(--colors-data-default-blue);
+        --data-visualization-coral: var(--colors-data-default-coral);
+        --data-visualization-jungle: var(--colors-data-default-jungle);
+        --data-visualization-lavender: var(--colors-data-default-lavender);
+        --data-visualization-lilac: var(--colors-data-default-lilac);
+        --data-visualization-peach: var(--colors-data-default-peach);
+        --data-visualization-spring: var(--colors-data-default-spring);
```

Comment:
gjerne legg til alle disse for de andre theme vi har og så de alltid har like variabler